Doc: Add liquid examples (#1300)
3scale/APIcast: doc/liquid_conditions.md

# Liquid conditions examples

> When using liquid, a lot of things can be achieved, and here are some examples

Reviewer: You can use Liquid to configure conditions applicable across multiple policies to <insert_purpose>

Some questions to help you:
* What's the benefit of using Liquid to configure conditions across multiple policies?
* Are there any prerequisites to use these examples?
* Are there any limitations with these examples? For instance, can these examples be used in a production environment?

of conditions that can be used across multiple policies

## Headers

### Header is not present

> Condition pass if header 'Backend' is sent and it's staging

Reviewer: ```suggestion
Condition passes if the  'Backend' header is sent and it's staging
```

Questions:
a. Who/What sends the  'Backend' header ?
b. What does `staging` refer to?

```
"condition":{
   "operations":[
      {
         "left":" {% if headers['Backend'] %}{{ headers['Backend'] }}{%else%}Notpresent{%endif%}",
         "left_type":"liquid",
         "op":"==",
         "right":"staging"
      }
   ]
}
```

> If want to test when the header is not present, the condition will be like this:

Reviewer: ```suggestion
If you want to test when the `Backend` header is not present, the condition is as follows:
```

```
"condition":{
   "operations":[
      {
         "left":" {% if headers['Backend'] %}IsPresent{%else%}Notpresent{%endif%}",
         "left_type":"liquid",
         "op":"==",
         "right":"Notpresent"
      }
   ]
}
```

## Numeric operations

> These are harder, by default, all the values on liquid are strings, so we need

Reviewer: ```suggestion
Numeric operations require additional steps. By default, all the values on liquid are strings, so you need
```

> to get any integer before do any operation. We can use `minus` or `plus`

Reviewer: ```suggestion
to get any integer before do any operation. Use `minus` or `plus`
```

> filters using 0, and do something like this:

Reviewer: ```suggestion
filters along with `0` (zero), and apply the following configuration:
```

```
"condition":{
  "operations":[
    {
      "left": "{% assign val = headers['version'] | minus:0 %} {%if val < 20 %}PASS{%else%}DENIED{%endif%}",
      "left_type":"liquid",
      "op":"==",
      "right":"PASS"
    }
 ]
}
```

@TODO: something like a int filter should be added.
@TODO: default filter should be added too.

> ## Another examples:

Reviewer: ```suggestion
## Additional examples:
```
